mtp_bank_admin/apps/bank_admin/tests/test_adi.py

```diff
@@ -44,7 +44,7 @@ def get_adi_transactions(type, count=20):
             else:
                 transaction['prison'] = TEST_PRISONS[2]
 
-        transaction['amount'] = Decimal(random.randint(500, 5000))/100
+        transaction['amount'] = Decimal(random.randint(500, 5000))
```

Comment:
I think this should return integer not decimal as the API would return integer values, no?